tools/SetupFlow/DevHome.SetupFlow/ViewModels/ConfigurationUnitResultViewModel.cs

```diff
@@ -147,5 +149,31 @@ private string GetUnitErrorMessage()
         }
 
         return _stringResource.GetLocalized(StringResourceKey.ConfigurationUnitFailed, resultCodeHex);
-    }
+    }
+
+    private string GetErrorDescription()
+    {
+        if (string.IsNullOrEmpty(_unitResult.ErrorDescription))
+        {
+            return string.Empty;
+        }
+
+        switch (_unitResult.HResult)
+        {
+            case WinGetConfigurationException.WingetConfigErrorDuplicateIdentifier:

```

Comment:
Should we add a comment, above these to say that these are WinGet specific errors and then a comment under the default case saying that this error is coming directly from the DSC resource itself?

Reply:
Agree comment(s) would be helpful, I don't think it's clear why all those errors don't return a string but the default case does.

Reply:
Good point! I got more context from winget team and added reference to the winget code where the logic has been inspired from.